README.md

```diff
@@ -20,7 +20,7 @@ Tika jars can be fetched from Maven Central or your favourite Maven mirror.
 
 **Tika 1.X reached End of Life (EOL) on September 30, 2022.**  
 
-Tika is based on **Java 8** and uses the [Maven 3](https://maven.apache.org) build system. 
+Tika is based on **Java 11** and uses the [Maven 3](https://maven.apache.org) build system. 
```

Comment:
Probably better in a separate commit, but let's leave it. Same goes for other improvements in this file.

Reply:
Oh, it was in a separate commit before, but the pull request template recommends squashing all the changes. I can easily split this if you want. :-)